src/xfce.yml

```
@@ -3,8 +3,10 @@
   name: Install Xfce
   become: yes
   become_method: sudo
-  roles:
-    - role: xfce_cool
+  tasks:
+    - name: Install XFCE for the COOL
```

Use the capitalization that we use above, which is also how they do it on their [own site](https://www.xfce.org/):
```suggestion
    - name: Install Xfce for the COOL
```